tests/reg_tests/test_DVConstraints.py

```diff
@@ -320,6 +320,53 @@ def test_thickness1D_box(self, train=False, refDeriv=False):
                 funcs["DVCon1_thickness_constraints_2"], 8.0 * np.ones(3), name="thickness_base", rtol=1e-7, atol=1e-7
             )
 
+    def test_projected_thickness1D_box(self, train=False, refDeriv=False):
```

Comment:
I wonder if we could use `parameterized` to avoid code duplication with the previous test

Reply:
Perhaps, but the skewing of the design variables (line 347 onward) is different enough that it wouldn't be trivial. If we add another thickness con it might be worth creating a common function like `self.genertate_dvgo_dvcon` that does some common thickness testing. But since it is only like 30 lines of duplicated code total, I think its probably fine.